atlasdb-client/src/main/proto/com/palantir/atlasdb/protos/TableMetadataPersistence.proto

```diff
@@ -12,6 +12,7 @@ message TableMetadata {
     optional SweepStrategy sweepStrategy = 9;
     optional int32 explicitCompressionBlockSizeKiloBytes = 10;
     optional bool appendHeavyAndReadLight = 11;
+    optional bool nameLoggable = 12 [default = false];
```

Comment:
Yep, that's right. I generally prefer to keep things optional where possible for backwards- (and forwards-) compatibility reasons.

Actually for AtlasDB we're probably OK given how we write the metadata table (in TM.create we write the metadata from the schema to a byte[], and replace the existing byte[] if ours is different), but I'd prefer not to rely on this behaviour.